onfido/exceptions.py

```diff
@@ -23,6 +23,9 @@ class OnfidoConnectionError(OnfidoError):
 class OnfidoTimeoutError(OnfidoError):
     pass
 
+class OnfidoRegionError(OnfidoError):
```

Comment:
Minor: I wouldn't subclass `OnfidoError` here in case clients have error handling for it. You could probably just subclass Exception, as it's not related to errors making a request like I think the others are.

Reply:
Thanks, good point, have changed